### PR TITLE
Improve iostat wrapper

### PR DESCRIPTION
**Pull request by mmichal10**
Signed-off-by: Michal Mielewczyk <michal.mielewczyk@intel.com>

### CODE REVIEW

File: test_tools/iostat.py [hunk 1]
```diff
@@ -5,53 +5,107 @@
...
-        self.total_writes = Size(float(stat_list[4]), Unit.KibiByte)
+    def __init__(self, device_name: str, raw_stats: list = None):
+        metrics_number = 13
+        if raw_stats is None:
+            raw_stats = [0] * metrics_number
+        if len(raw_stats) < metrics_number:
+            raw_stats.extend([0] * metrics_number)
+
+        self.device_name = device_name
+        self.read_requests_merged_per_second = float(raw_stats[0])
```
Comment (katlapinka) [inline test_tools/iostat.py:21]: shouldn't it be Size, as write_requests_merged_per second below?

File: test_tools/iostat.py [hunk 2]
```diff
@@ -5,53 +5,107 @@
...
+            f"Write requests: {self.write_requests}\n"
+            f"Reads per second: {self.reads_per_sec}\n"
+            f"Writes per second {self.writes_per_sec}\n"
+            f"Average request size: {self.average_request_size} blocks\n"
+            f"Average queue length {self.average_queue_length}\n"
+            f"Average service time: {self.average_service_time}\n"
+            f"Read average service time {self.read_average_service_time}\n"
+            f"Write average service time: {self.write_average_service_time}\n"
+            f"Utilization: {self.utilization}\n"
+            f"=================================================\n"
```
Comment (Ostrokrzew) [inline test_tools/iostat.py:50]: If You want, You can change it to triple-quoted fstring.

File: test_tools/iostat.py [hunk 3]
```diff
@@ -5,53 +5,107 @@
...
-            f"| grep {device.system_path.strip('/dev/')}").stdout.splitlines()
-        iostat_list = []
+            f"{iostat_cmd} " f'| grep "{grep_pattern}"'
+        ).stdout.splitlines()
+
+        ret = []
+        raw_stats = []
         for line in lines:
             args = line.split()
-            if args[0] == device.system_path.strip('/dev/'):
```
Comment (katlapinka) [inline test_tools/iostat.py:54]: Why did you remove this if? What if the first element of line will be different than device name? e.g. some header in the first line

File: test_tools/iostat.py (hunk 1 above)
Comment (mmichal10) [reply test_tools/iostat.py:21]: Actually `write_requests_merged_per` should be raw float, but thanks for noticing it

File: test_tools/iostat.py (hunk 3 above)
Comment (mmichal10) [reply test_tools/iostat.py:54]: Since iostat output is filtered with `grep`, headers won't be included.
